battleship.py

```diff
@@ -163,17 +183,32 @@ def isHorizontal(ship):
 Returns: list of ints
 '''
 def getClickedCell(data, event):
-    return
+    x = int(event.y/data["cell"])
+    y = int(event.x/data["cell"])
+    clickedCell = []
+    clickedCell.append(x)
+    clickedCell.append(y)
+    return clickedCell
 
 
 '''
 drawShip(data, canvas, ship)
 Parameters: dict mapping strs to values ; Tkinter canvas; 2D list of ints
 Returns: None
-'''
+
 def drawShip(data, canvas, ship):
     return
+'''    
 
+def drawShip(data, canvas, ship):
+    cellsize = data["cell"]
+    for row in range(data["rows"]):
+        for col in range(data["cols"]):
+            grid = [row,col]

```

Comment:
Use some other variable name instead of grid, like cell or position

Reply:
@Pragya28 Changed the variable name. You can review.